index.html

```diff
@@ -19,7 +19,7 @@
 	<body>
 		<main>
 			<div class="content">
-				<p class="todays-motivation">Today's Motivation</p>
+				<h5 class="todays-motivation">Today's Motivation</h5>
```

Comment:
The semantics aren't here still. You can't use an h5 without there being an h1,h2,h3,h4 already on the page.